src/components/modals/GameRunningModal.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <div id="gameRunningModal" :class="['modal', {'is-active': isOpen}]">
+        <div class="modal-background" @click="close"></div>
+        <div class="modal-content">
+            <div class='notification is-info'>
+                <h3 class="title" v-if="isSteamGame">{{ activeGame.displayName }} is launching via Steam</h3>
+                <h3 class="title" v-else>{{ activeGame.displayName }} is starting</h3>
+                <h5 class="title is-5">Close this message to continue modding.</h5>
+                <div v-if="isSteamGame">
+                    <p>If this is taking a while, it's likely due to Steam starting.</p>
+                    <p>Please be patient, and have fun!</p>
+                </div>
+            </div>
+        </div>
+        <button class="modal-close is-large" aria-label="close" @click="close"></button>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from "vue-property-decorator";
+
+import GameManager from "../../model/game/GameManager";
+import { StorePlatform } from "../../model/game/StorePlatform";
+
+@Component
+export default class GameRunningModal extends Vue {
+    activeGame = GameManager.activeGame;
```

Comment:
Does this change whenever the game is changed via settings?
As far as I'm aware it didn't and that's why binds are in created hooks instead, could be wrong though.

Reply:
It does change, since the modal gets recreated/remounted every time the user travels to `GameSelectionScreen` and returns to `Manager` screen, and when it's created the value of `activeGame` has already been updated by other components. If we ever change it so the game can be reselected directly from the `Manager` screen, we probably need to rethink this as well.

Reply:
We could also decouple the component entirely from the `GameManager` and simply pass in the active game name as a prop to the component. I'd feel most comfortable with that solution as there's less coupling that way.